api/authentication/views.py

```diff
@@ -54,23 +53,23 @@ def get(self, request, *args, **kwargs):
                 return Response(serializer.data, status=status.HTTP_200_OK)
         except StdUser.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
-    
+
     def post(self, request, *args, **kwargs):
         return Response(status=status.HTTP_405_METHOD_NOT_ALLOWED)
-    
-    @permissions(["IsModeratorUser", "IsUser"])
+
+    @permissions(["IsModeratorUser", "IsUser"],"kwargs")
     def put(self, request, *args, **kwargs):
         number = args[0]
         queryset = User.objects.filter(id=number.get('id'))
         self.serializer_class = UpdateUserSerializer
         serializer = self.serializer_class(queryset[0],  data=request.data)
-        
+
         if(serializer.is_valid()):
             serializer.save()
-        
+
         return Response(status=status.HTTP_200_OK)
-        
-    @permissions(["IsModeratorUser", "IsUser"])
+
+    @permissions(["IsModeratorUser", "IsUser"],"kwargs")
     def delete(self, request, *args, **kwargs):
```

Comment:
In decorator you're using kwargs, so args cause error. Need to be fixed. You have to use only kwargs variable